### DISCUSSION TITLE
Separate the first two pages of the report

### DISCUSSION THREAD

**Discussion by dansyndicate** [joaomlourenco/novathesis#477] [Q&A] [answered]
Good evening, Professor,

Could you tell me how I can separate the first two pages of the dissertation preparation document, because it will be better this way for when I print it.

And if you also know how to do the same to separate the content index, images and tables with a blank page between them?

Compliments,
Daniel Martins

![image](https://github.com/joaomlourenco/novathesis/assets/129518837/5e72c128-e5bf-4901-8e12-296de718218b)

**Comment by joaomlourenco** [comment]
Tens a dissertação em "modo screen". 
Descomenta a linha
```latex
% \ntsetup{media=paper}
```
em `Config/1_novathesis.tex`.  Deve fazer o que pretendes.

**Reply by dansyndicate** [reply]
I did that but it only worked for the titles of chapters, the first two pages (covers) they are still next to each other

![image](https://github.com/joaomlourenco/novathesis/assets/129518837/dc1f69a8-4014-4260-9d91-85fb6d87e2f8)

**Comment by joaomlourenco** [comment]
That's a bug. Let me check…

**Reply by dansyndicate** [reply]
Ok professor thank you for your help. 
I will wait for your answer.

**Reply by joaomlourenco** [reply, marked as answer]
You may download the new `novathtesis.cls` file to replace the one you have.  It should be ok.  Please let me know.

**Reply by dansyndicate** [reply]
Yes professor it worked perfectly.
